src/_selectors/AssetIndexSelectors.js

```diff
@@ -23,7 +24,32 @@ export const shownAssetIndexRowsSelector = createSelector(
     }
 );
 
-export const assetIndexHeadersSelector = createSelector(
+const assetIndexRowToDuration = row =>
+    row ? `${row[2]}–${row[3]}` : '—';
+
+const assetIndexTableSelector = createSelector(
+    shownAssetIndexRowsSelector,
```

Comment:
It will be optional, but will hook on data received and will fire the events wit the transformed data. 'Data Transform Middleware' :)